packages/coding-standard/src/Fixer/Annotation/RemovePHPStormAnnotationFixer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Fixer\Annotation;
+
+use Nette\Utils\Strings;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Tokens;
+use SplFileInfo;
+use Symplify\CodingStandard\Fixer\AbstractSymplifyFixer;
+
+/**
+ * @see \Symplify\CodingStandard\Tests\Fixer\Annotation\RemovePHPStormAnnotationFixer\RemovePHPStormAnnotationFixerTest
+ */
+final class RemovePHPStormAnnotationFixer extends AbstractSymplifyFixer
+{
+    /**
+     * @see https://regex101.com/r/nGZBzj/2
+     * @var string
+     */
+    private const CRETED_BY_PHPSTORM_DOC_REGEX = '#\/\*\*\s+\*\s+Created by PHPStorm(.*?)\*\/#msi';
```

Comment:
Can we fit new, or rather old comment style to this fixer?
```
/**
 * Created by JetBrains PhpStorm.
 * User: user
 * Date: 7/9/13
 * Time: 5:29 PM
 * To change this template use File | Settings | File Templates.
 */
```

Reply:
Sure :+1: please send PR with new regex and test